events/wc_pandaria_events.txt

```diff
@@ -78,22 +76,349 @@ narrative_event = {
 				capital_scope = { region = world_pandaria }
 			}
 		}
-		name = EVTOPTA_WCPAN_100_expeditions
+		name = EVTOPTA_WCPND_100_expeditions
+	}
+}
+
+### ANKOAN COURTIERS ###
+
+# Ankoan courtier appears at court
+character_event = {
+	id = WCPND.200
+	desc = EVTDESC_WCPND_200
+	picture = GFX_evt_emissary #PH
+	border = GFX_event_normal_frame_diplomacy
+	
+	min_age = 16
+	prisoner = no
+	only_playable = yes
+	capable_only = yes
+	only_independent = yes
+	
+	is_triggered_only = yes
+	
+	trigger = {		
+		OR = {
+			religion = waterspeaker
+			religion = water_deities
+			religion = neptulon_worship
+		}
+		NOT = { 
+			culture = ankoan
+			is_incapable = yes
+			is_inaccessible_trigger = yes
+		}
+		# Can only appear after the Naga have spawned in the world
+		has_global_flag = naga_resurfaced_flag
+	}
+	
+	# More likely to appear in Tidehunter courts
+	weight_multiplier = {
+		days = 1
+		modifier = {
+			factor = 1.5
+			religion = neptulon_worship
+		}
+	}
+	
+	immediate = {
+		create_character = {
+			random_traits = yes
+			culture = ankoan
+			dynasty = culture
+			religion = neptulon_worship
+			female = no
+			age = 30
+			flag = ai_flag_refuse_conversion
+		}
+		new_character = {
+			remove_lifestyle_trait_effect = yes
+			random_list = {
+				5 = {
+					add_trait = socializer
+				}
+				5 = {
+					add_trait = socializer
+					add_trait = shrewd
+				}
+				5 = {
+					add_trait = administrator
+				}
+				5 = {
+					add_trait = administrator
+					add_trait = shrewd
+				}
+				10 = {
+					add_trait = schemer
+					add_trait = class_rogue_3
```

Comment:
Shouldn't they be stronger? The 3rd level is below average (the 5th).

Reply:
changed to 4-5